src/utils/exit-fullscreen.js

```diff
@@ -1,4 +1,8 @@
 export default (() => {
+  if (typeof document === 'undefined') {
+    return () => {}
```

Comment:
I've decided to go with early return and no-op function here, because this function still expected to return some function anyway. Without `document` it is reasonable, that the resulting function could be simply empty